src/dotnet/Chat.UI.Blazor/Services/ChatUI.cs

```diff
@@ -42,17 +53,20 @@ public ChatUI(IServiceProvider services)
         var localSettings = services.GetRequiredService<LocalSettings>().WithPrefix(nameof(ChatUI));
         var accountSettings = services.GetRequiredService<AccountSettings>().WithPrefix(nameof(ChatUI));
         ActiveChatId = StateFactory.NewKvasStored<Symbol>(new(localSettings, nameof(ActiveChatId)));
-        PinnedChatIds = StateFactory.NewKvasSynced<ImmutableDictionary<string, Moment>>(
+        _pinnedChatIds = StateFactory.NewKvasSynced<ImmutableDictionary<string, Moment>>(
             new(accountSettings, nameof(PinnedChatIds)) {
                 InitialValue = ImmutableDictionary<string, Moment>.Empty,
                 Corrector = FixPinnedChatIds,
             });
-        RecordingChatId = StateFactory.NewKvasStored<Symbol>(new(localSettings, nameof(RecordingChatId)));
-        ListeningChatIds = StateFactory.NewKvasStored<ImmutableList<Symbol>>(
+        _recordingChatId = StateFactory.NewKvasStored<Symbol>(new(localSettings, nameof(RecordingChatId)));
+        _listeningChatIds = StateFactory.NewKvasStored<ImmutableList<Symbol>>(
             new(localSettings, nameof(ListeningChatIds)) {
                 InitialValue = ImmutableList<Symbol>.Empty,
                 Corrector = FixListeningChatIds,
             });
+        _activeChatIds = StateFactory.NewMutable(ImmutableList<(Symbol, Moment)>.Empty);
```

Comment:
I think we should use some synced state there and store active chats at the server - we would like to have the synced across all devices

Reply:
done